Can't build on Linux

I cloned the repository using git. 
Trying to run the suggested build commands the build fails:
```
# cmake -S . -B build
-- The CXX compiler identification is GNU 10.2.1
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for C++ include pthread.h
-- Looking for C++ include pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE  
-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libOpenGL.so   
CMake Error at CMakeLists.txt:30 (add_subdirectory):
  The source directory

    /home/tomi/git/AgIsoDDOPGenerator/submodules/agisostack

  does not contain a CMakeLists.txt file.


CMake Error at CMakeLists.txt:31 (add_subdirectory):
  The source directory

    /home/tomi/git/AgIsoDDOPGenerator/submodules/sdl

  does not contain a CMakeLists.txt file.


-- Configuring incomplete, errors occurred!
See also "/home/tomi/git/AgIsoDDOPGenerator/build/CMakeFiles/CMakeOutput.log".
See also "/home/tomi/git/AgIsoDDOPGenerator/build/CMakeFiles/CMakeError.log".
```
Looks like I'm missing something important. What's going wrong here and how can I get it to build?

--- a/README.md
+++ b/README.md
@@ -24,12 +24,19 @@ This project is built with CMake.
 Make sure you have all the dependencies installed.
 
 ```
-sudo apt install cmake build-essential
+sudo apt install cmake build-essential libgl1-mesa-dev
+```
+
+Clone the repo:
+
+```
+git clone https://github.com/Open-Agriculture/AgIsoDDOPGenerator.git --recurse-submodules
 ```
 
 Then compile with CMake:
 
 ```
+cd AgIsoDDOPGenerator
 cmake -S . -B build
 cmake --build build
 ```

